content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx

```diff
@@ -13,11 +13,11 @@ This page helps you make an informed decision on when and how to upgrade from Pr
 The upgrade documentation consists of several pages, here's an overview of how to use them:
 
 - **How to upgrade** (_you are here_): Starting point to learn about the upgrade process in general.
-- [Schema incompatibilities](./schema-incompatibilities-postgres): A _reference_ page about the schema incompatibilities between Prisma 1 and Prisma 2. Reading this page is optional but it will give you a better understanding of certain steps in the upgrade process.
+- [Schema incompatibilities](/guides/upgrade-guides/upgrade-from-prisma-1/schema-incompatibilities): A _reference_ page about the schema incompatibilities between Prisma 1 and Prisma 2. Reading this page is optional but it will give you a better understanding of certain steps in the upgrade process.
```

Comment:
Is that link correct? I think the target has a tech switcher.

Reply:
Oh good catch!